src/suricata-plugin.h

```diff
@@ -53,7 +53,7 @@ typedef struct SCCapturePlugin_ {
 int SCPluginRegisterCapture(SCCapturePlugin *);
 
 // Every change in the API used by plugins should change this number
-#define SC_PLUGIN_API_VERSION 8
+#define SC_PLUGIN_API_VERSION 0x801
```

Comment:
I still don't think we should be twiddling this for a development release.

For 8.0, we should use something like 800000, or something.. 9.0, 900000.  If we did an 8.1 then 800100.

But we are going to twiddle it, just start at 1.. Then 2, then 3, etc..

Reply:
I think we should be twiddling this for a development release, for the people using `git master` so that they can use plugins easily after 8.0 release

Ok to increase it 1 by 1